float64 angularVelocity
float64 LineHeadingError
float64 headingErr

